src/remote/oc_remote_options.lsl

```diff
@@ -219,7 +219,7 @@ DefinePosition() {
     integer iPosition = llListFindList(g_lAttachPoints, [llGetAttached()]);
     vector size = llGetScale();
 //  Allows manual repositioning, without resetting it, if needed
-    if (iPosition != g_iSPosition && iPosition != -1) { //do this only when attached to the hud 
+    if (iPosition != g_iSPosition) {
```

Comment:
can we get this change back please, else the hud will move around each time you work on it when rezzed on the ground